nodeup/pkg/model/kubelet.go

```diff
@@ -255,6 +255,9 @@ func buildKubeletComponentConfig(kubeletConfig *kops.KubeletConfigSpec, provider
 	if kubeletConfig.ImageMinimumGCAge != nil {
 		componentConfig.ImageMinimumGCAge = *kubeletConfig.ImageMinimumGCAge
 	}
+	if kubeletConfig.MaxParallelImagePulls != nil {
+		componentConfig.MaxParallelImagePulls = kubeletConfig.MaxParallelImagePulls
+	}
```

Comment:
I don't think there's any harm in just doing the assignment. If `kubeletConfig.MaxParallelImagePulls=nil`, then `componentConfig.MaxParallelImagePulls` will remain at its default value.
```suggestion
	componentConfig.MaxParallelImagePulls = kubeletConfig.MaxParallelImagePulls
```